tasks/9_find-numbers/find-numbers.js

```diff
@@ -0,0 +1,3 @@
+export const getNumbers = (string) => {
+  return string.match(/\d+/g).map((item) => parseInt(item));
```

Comment:
That's really interesting. Function **parseInt(string)** will convert a string containing non-numeric characters to a number, as long as the string begins with numeric characters, but function **Number(string)** will return **NaN** if the string contains any non-numeric characters. Since I used **RegEx** to match all numbers with `\d+`, this will work for all integer values. Actually, I can write it even shorter: `return string.match(/\d+/g).map(Number)`.